packages/engine-test-utils/src/__tests__/engine-server/markdown/noteRefv2.spec.ts

```diff
@@ -333,7 +329,7 @@ describe("noteRefV2", () => {
       );
     });
 
-    describe("AND note has beginning section", () => {
+    describe.skip("AND note has beginning section", () => {
```

Comment:
Hey @kevinslin, need help with this test case. What is the significance of `^end`  in `![[foo#header-1^end]]` and how does it differ from `![[foo#header-1]]` when `enableSmartRefs` is false.

Reply:
it grabs everything from `#header-1` to the very last line of the doc. it shoudl have the same behavior as `![[foo#header-1]]` when `enableSmartRefs` is false (this is so that users can re-create the original range using smart refs)